docs/changehistory/NextVersion.md

```diff
@@ -10,6 +10,8 @@ Table of contents:
   - [Fixes](#fixes-1)
 - [@itwin/components-react](#itwincomponents-react)
   - [Fixes](#fixes-2)
+- [@itwin/appui-layout-react](#itwinappui-layout-react)
+  - [Changes](#changes)
```

Comment:
@GerardasB As this package is internal, should we have NextVersion for it ?

Reply:
I'd say no and put the changes under appui-react.

Reply:
Moved to appui-react.